### PR TITLE
OD-423 [Fix]  Added support for IE11 and Safari to render timePicker

### PR DESCRIPTION
**Pull request by AndrRyaz**
@sofiiakvasnevska 

## Issue

OD-423 https://weboo.atlassian.net/browse/OD-423

## Description
Added support of this feature for IE11 and Safari
## Screenshots/screencasts
http://somup.com/crf12tbK01
## Backward compatibility
This change is fully backward compatible.
## Reviewers
@upplabs-alex-levchenko @ivandevupp @YaroslavOvdii

### CODE REVIEW

File: js/components/time.js [hunk 1]
```diff
@@ -23,6 +23,8 @@ Fliplet.FormBuilder.field('time', {
   },
   data: function() {
     return {
+      timepicker: null,
+      LOCAL_FORMAT: moment.localeData().longDateFormat('LT'),
```
Comment (squallstar) [inline js/components/time.js:27]: I already reviewed this in a different PR. Please open this PR without these changes once the other is merged.

**Review by squallstar — CHANGES_REQUESTED** [review]
Rejected +1

**Comment by AndrRyaz** [comment]
@squallstar, I already fixed this, needed to update on master branch

**Review by squallstar — APPROVED** [review]
@sofiiakvasnevska this is ready to be deployed on **staging** for QA.

- **Risk**: High
- **Affected**: Forms with date fields
- **Verify**: Verify forms with date fields work as expected

**Pull request merged**